glean_parser/lint.py

```diff
@@ -130,16 +150,41 @@ def check_bug_number(metric):
         )
 
 
+def check_valid_in_baseline(metric, parser_config={}):
```

Comment:
Should we merge #123 first?

Reply:
Oh yes, forgot that I based it on that branch